src/main/resources/application.properties

```diff
@@ -1,3 +1,3 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/
+spring.datasource.url=jdbc:mysql://localhost:3306/food_project
```

Comment:
add this to the end of the line
`?createDatabaseIfNotExist=true`
so the whole url will look like this:
`jdbc:mysql://localhost:3306/food_project?createDatabaseIfNotExist=true`
This way the application will create the MySQL database(`food_project`) if it does not exist.